tests/dummy/app/app.js

```diff
@@ -1,6 +1,6 @@
 import Ember from 'ember'
 const {Application} = Ember
-import Resolver from 'ember-resolver'
+import Resolver from './resolver'
```

Comment:
Why are we proxying through a module that just exports the resolver from `ember-resolver`?